core/src/main/java/hudson/Functions.java

```diff
@@ -612,7 +612,7 @@ public static String getCookie(HttpServletRequest req, String name, String defau
     private static final Pattern ICON_SIZE = Pattern.compile("\\d+x\\d+");
 
     @Restricted(NoExternalUse.class)
-    public static String validateIconSize(String iconSize) throws SecurityException {
+    public static String validateIconSize(String iconSize) {
```

Comment:
[Effective Java 3rd Edition §74](https://books.google.com/books?id=BIpDDwAAQBAJ) says:

> Use the Javadoc `@throws` tag to document each exception that a method can throw, but do not use the `throws` keyword on unchecked exceptions.

This PR removes the `throws` keyword on this unchecked exception (good), but it does not add the Javadoc `@throws` tag (bad).